pandas/tests/plotting/frame/test_frame.py

```diff
@@ -603,11 +603,11 @@ def test_area_lim(self, stacked):
         lines = ax.get_lines()
         assert xmin <= lines[0].get_data()[0][0]
         assert xmax >= lines[0].get_data()[0][-1]
-        assert ymin == 0
+        assert ymin == 0, ymin
```

Comment:
Was this intended to be included in this PR?

Reply:
Whoops - no, that was for another PR. Thanks, reverted.